controllers/volsync/vshandler.go

```diff
@@ -521,22 +521,6 @@ func (v *VSHandler) TakePVCOwnership(pvcName string) (bool, error) {
 		return false, err
```

Comment:
The documentation for this function says:

> // and then will remove ACM annotations and also add VRG as the owner.  This is to break the connection between

But now we don't remove the ACM annotation.

Reply:
We'll change the documentation

Reply:
done